README.md

```diff
@@ -153,10 +162,13 @@ disables three-finger tapping. Defaults to 0.
 
 **ClickTime** - 
 When tapping, how much time to hold down the emulated button. Integer value
-representing milliseconds. Defaults to 50.
+representing milliseconds. Not very usable because button release event will be
+sent immediately after releasing any of fingers (touches).
+Integer value representing miliseconds. Defaults to 50.
 
 **MaxTapTime** - 
-The amount of time to wait for a tap to release before counting it as a move.
+The amount of time to wait for incoming touches after first one before counting
+it as emulated button click.
```

Comment:
Your new statement is less clear than the previous one. If you're going to rewrite it then it would be better to keep the old explanation or write one that is clearer. In both cases an example would be add much.